cookbooks/swift/recipes/source.rb

```diff
@@ -2,8 +2,13 @@
 #Copyright (c) 2015-2021, NVIDIA CORPORATION.
 #SPDX-License-Identifier: Apache-2.0
 
-# ensure source_root
+if node['use_python3']
+  default_pip = 'pip3'
+else
+  default_pip = 'pip2'
+end
```

Comment:
Yeah, placed it in every chef file because I didn't know how later chef runs (non porovisioner) would hadn'e things. And hadn't got around to debugging it yet. (also attempting to re-learn chef) :P